easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-foss-2015b.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.14'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+perl = 'Perl'
+perlver = '5.20.2'
+perlsuffix = '-bare'
+dependencies = [
+    (perl, perlver, perlsuffix),
```

Comment:
We typically try to use the latest version for dependencies too.

The other thing to keep in mind is that the modules that get installed may need to be used together with other modules. So, some effort to try and avoid possible conflicts is good.

It's impossible to avoid all conflicts though, but not using a `-bare` version gives more options.
We really only use `-bare` versions as a build dependency (since a full-blown install with extensions is typically not needed then).